### ISSUE TITLE
Crash (BAD_ACCESS) on iOS when using 64bit architecture

### ISSUE THREAD

**Issue by jeremyfa** [HaxeFlixel/flixel#1411]
### How to reproduce the issue

I reproduced the issue with the HelloWorld example using the latest flixel, openfl and lime versions.

Just export a new xCode project from the HelloWorld example using ``lime update ios -debug`` and then build for a device that will use the 64bit binary (for example you can choose iPhone 6 Plus if you run a simulator, or a real iOS device that can run 64bit. I tested on a retina iPad mini).

The project seems to build successfully but crashes when calling ``setupGame``.

![capture d ecran 2015-01-02 a 15 17 51](https://cloud.githubusercontent.com/assets/164094/5596640/c6d33f60-9293-11e4-9e9e-9002fc54cd2c.png)

The project, however, works perfectly fine if I disable 64bit architecture and only use armv7, or if I run it on a device that cannot run arm64.

I am using the latest xCode on OS X Mavericks.

### Dump of the library versions I use

```
flixel-addons: [1.1.0]
flixel-demos: [1.1.1]
flixel-templates: [1.0.2]
flixel-tools: [1.0.3]
flixel-ui: [1.0.2]
flixel: [3.3.6]
hxcpp: [3.1.48]
lime: [2.0.4]
openfl-samples: [2.2.0]
openfl: [2.2.1]
swf: [1.7.3]
```

**Comment by jeremyfa** [comment]
I recently changed computer, moved to OS X Yosemite with a super clean install, and the issue is still present. Anybody having the same problem ? Starting from february 1st, this issue may become a real problem as Apple will not accept apps that are not built in 64bit.

**Comment by sruloart** [comment]
Is this a HaxeFlixel issue? this recent OpenFL [thread](http://community.openfl.org/t/64-bit-upgrade-to-support-feb1st-req-for-ios-app-store/279/15) seems relevant. Try working with the git branches (openfl/lime).

**Comment by jeremyfa** [comment]
Not sure it is a HaxeFlixel issue. However I know the crash is happening inside some HaxeFlixel code, but at the same time it is working fine with armv7. I already referenced this issue on the lime repository, but I will also check this thread.

**Comment by jeremyfa** [comment]
Some updates on this issue: I installed latest lime, openfl, hxcpp from their development repositories, and rebuilt all the binaries. The issue disappeared. I suspect that something was not built correctly on either lime or openfl's release versions, or that something has been fixed on hxcpp recently, but I don't really know. Well, it seems like it was not a flixel issue. I will keep using the development versions for now!
